File: outfit-planner-system/outfit-service/src/main/java/com/outfit/planner/system/outfit/service/dataaccess/outfithistory/entity/WeatherConditions.java
package com.outfit.planner.system.outfit.service.dataaccess.outfithistory.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WeatherConditions {
    private String temperature;
    private String windSpeed;
    private String weatherCode;
}

File: outfit-planner-system/prediction-service/app/model/weather_conditions.py
class WeatherConditions:

  def __init__(self, temperature, weatherCode, windSpeed):
    self.temperature = temperature
    self.weatherCode = weatherCode
    self.windSpeed = windSpeed

  @staticmethod
  def from_json(json):
    return WeatherConditions(json['temperature'], json['weatherCode'], json['windSpeed'])
